ext/npm-extension.js

```diff
@@ -389,6 +389,25 @@ exports.addExtension = function(System){
 		oldConfig.apply(loader, arguments);
 	};
 
+	if (typeof steal !== undefined) {
+		steal.addNpmPackages = function(packages) {
+			packages = packages || [];
+
+			for (var pkg of packages) {
```

Comment:
![75824661](https://cloud.githubusercontent.com/assets/724877/23322636/1c5410c2-faa2-11e6-9bb6-3a050799d9b1.jpg)

working on it